repository-data/webfiles/src/main/resources/site/freemarker/common/facets.ftlh

```diff
@@ -1,6 +1,5 @@
 <#include "../include/imports.ftlh">
-<@hst.setBundle basename="month-names"/>
-<@hst.setBundle basename="document-types"/>
+<@hst.setBundle basename="document-types,month-names"/>
```

Comment:
Suggest adding a comment to make it clear that we need to do a single setBundle only

Reply:
Good idea